src/test/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/service/ImportServiceTest.kt

```diff
@@ -184,11 +159,32 @@ internal class ImportServiceTest {
   @Test
   fun `given an import date of current or future date an exception is thrown when importing people and profiles`() {
     assertThatThrownBy {
-      importService.importPeopleProfiles(timeSourceWithFixedDate.date())
+      importReportsService.importPeopleProfileReportsStartingFrom(timeSourceWithFixedDate.date())
     }.isInstanceOf(RuntimeException::class.java)
 
     assertThatThrownBy {
-      importService.importPeopleProfiles(timeSourceWithFixedDate.date().plusDays(1))
+      importReportsService.importPeopleProfileReportsStartingFrom(timeSourceWithFixedDate.date().plusDays(1))
     }.isInstanceOf(RuntimeException::class.java)
   }
+
+  private class FakeReportImporter : ReportImporter(mock(), mock(), FakeReportReaderParser()) {
```

Comment:
This is bit ugly really but it was getting quite messy with all the mocking (still is a bit too be honest).